eip-draft_TCT_abbrev.md

```diff
@@ -0,0 +1,220 @@
+---
```

Comment:
```suggestion
---
eip: 7534
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
what do you mean to update the filename? I give the name as the specification: <eip-draft-name-abbrev>.md. Do you mean change the eip to 7534?